### PR TITLE
fix wrong output title for docker volume ls

### PR DESCRIPTION
**Pull request by lixiaobing1**
fix wrong output titile for docker volume ls

### CODE REVIEW

File: docs/extend/index.md [hunk 1]
```diff
@@ -97,7 +97,7 @@ started without error.
     ```bash
     $ docker volume ls
 
-    DRIVER              NAME
+    DRIVER              VOLUME NAME
     vieux/sshfs         sshvolume
```
Comment (lixiaobing10051267) [inline docs/extend/index.md]: from test, command "docker volume ls" output “VOLUME NAME”, so it should be fixed all "NAME" to "VOLUME NAME", is that right?

Comment (AkihiroSuda) [inline docs/extend/index.md]: Perhaps in https://github.com/docker/docker/commit/a488ad1a090eb0b6069f79cd045badaaa8ec1bda

Comment (vdemeester) [inline docs/extend/index.md]: ohh :sweat: This is a small regression on master then :sweat: 
/cc @thaJeztah should we put `VOLUME NAME` back ?

Comment (thaJeztah) [inline docs/extend/index.md]: @vdemeester oh, yes; the change wasn't intentional, so if you can change it back, that'd be good (I _do_ like just "name", but it should be discussed first if we want to change)

Comment (vdemeester) [inline docs/extend/index.md]: PR on its way :angel:

**Review by thaJeztah — APPROVED** [review]
https://github.com/docker/docker/pull/27457 was merged

LGTM

**Pull request merged**